src/controller/java/CHIPDeviceController-JNI.cpp

```diff
@@ -374,7 +374,11 @@ JNI_METHOD(jboolean, isConnected)(JNIEnv * env, jobject self, jlong deviceContro
 {
     ChipLogProgress(Controller, "isConnected() called");
     ChipDeviceController * deviceController = (ChipDeviceController *) deviceControllerPtr;
-    return deviceController->IsConnected() ? JNI_TRUE : JNI_FALSE;
+    if (deviceController->IsConnected())
```

Comment:
What error is this?

Reply:
Similar than the previous case for this file. It seems like doxygen parser is confused. It tells me that it can find the definition for `return deviceController IsConnected`which does not makes any sense.